.changeset/friendly-pillows-cough.md

```diff
@@ -0,0 +1,5 @@
+---
+'@clerk/backend': patch
+---
+
+Respect CloudFront-Forwarded-Proto header for pro
```

Comment:
Can we update the changeset here to align with our updated guidelines? https://github.com/clerkinc/javascript/blob/main/docs/CONTRIBUTING.md#changesets

Reply:
```suggestion
Update `authenticateRequest()` to respect the `CloudFront-Forwarded-Proto` header when determining the correct `forwardedProto` value. This fixes an issue when Clerk is used in applications that are deployed behind AWS CloudFront, where previously all requests were treated as cross-origin.
```

Reply:
@BRKalow changeset updated ptal.